healthyregions/SDOHPlace
# #382, #375: update details panel to add corresponding fields

This PR addresses issues #382 and #375. It aims to add featured variables to the detail panel, switch "subject" to "keywords," and update method names.

## How to Test
1. Navigate to the search page and review the list. The term "subject" should be replaced with "keywords" in both the result card and the detail panel:
<img width="1703" alt="Screenshot 2024-11-15 at 12 03 20 PM" src="https://github.com/user-attachments/assets/5157acdd-45c4-4d0e-adc4-4bbf58338bf6">

2. Search for "CDC Social Vulnerability Index."

3. Verify that the "featured variables" field is displayed, and the Methods/Data variable names have been updated:
<img width="1058" alt="Screenshot 2024-11-15 at 12 01 10 PM" src="https://github.com/user-attachments/assets/5ca81ca9-9a12-4cf9-a5e3-a95520cd1528">
